tests/integration/tier0/backup-release/test_backup_release.py

```diff
@@ -1,24 +1,55 @@
 import os
-import platform
+
+import pytest
 
 from conftest import SATELLITE_PKG_DST, SATELLITE_PKG_URL
 from envparse import env
 
 
-system = platform.platform()
+def assign_repository_variable(get_system_release, repository=None):
+    """
+    Helper function.
+    Assign correct repository setup to install subscription manager from to their respective distribution.
+    """
+
+    epel7_repository = "ubi"
+    epel8_repository = "baseos"
+    if "oracle-7" in get_system_release or "centos-7" in get_system_release:
+        repository = epel7_repository
+    elif "oracle-8" in get_system_release or "centos-8" in get_system_release:
+        repository = epel8_repository
+
+    return repository
+
+
+BACKUP_DIR = "/tmp/test-backup-release_backup/"
+BACKUP_DIR_EUS = "/tmp/test-backup-release_backup_eus/"
```

Comment:
Also, could be like that... Since the path is the same, we just change the `eus` part at the end of the first string. 

```suggestion
BACKUP_DIR_EUS = "%s_eus/" % BACKUP_DIR
```